src/APIClient/GPU.cpp

```diff
@@ -0,0 +1,16 @@
+#include <APIClient/GPU.hpp>
+
+APIClient::JSONResponse APIClient::GPU::putGpu(const nlohmann::json& vulkanInfosJson) {
+    nlohmann::json reponseJson;
+    RestClient::Response r = RestClient::put(APIClient::Router::getUrlString(Router::Route::putGpu), "application/json;charset=utf-8", vulkanInfosJson.dump());
+    reponseJson = nlohmann::json(r.body);
+    return std::make_tuple(r.code, reponseJson);
```

Comment:
So, you have `getRequestResponse`, why not doing a `putRequestResponse` for a future use?